package/src/test.rs

```diff
@@ -238,7 +248,7 @@ fn test_pants_bin_name_handling(scie_pants_scie: &Path) {
         false,
         r#"
             [GLOBAL]
-            pants_version = "2.15.0rc5"
+            pants_version = "2.18.0"
```

Comment:
By using the same pants version consistently, we avoid having to download and bootstrap more versions than necessary.